packages/protocol-sdk/src/premint/premint-client.ts

```diff
@@ -181,6 +176,17 @@ export class PremintClient extends ClientBase {
     return zoraCreatorFixedPriceSaleStrategyAddress[999];
   }
 
+  getDataFromPremintReceipt(receipt: TransactionReceipt) {
```

Comment:
better name would be `parsePremintTxReceipt` (adding the word `data` to the function name doesnt give any more info about what it's returning)